pkg/api/types.go

```diff
@@ -1202,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
+	// Fs represents a filesystem-based local volume
+	// +optional
+	Fs *LocalFsVolume
```

Comment:
Path to a block device makes sense, but we're not sure yet if there may be any other additional fields required.  So we prefer to have two different types to make field expansion easier in the future.

We are looking at exposing PVs as block devices in the future, so I think having that type field in the PV would make more sense.